### PR TITLE
docs: fix comment of prev_page()

### PR DESCRIPTION
**Pull request by donhui**
It shoud be `prev`, not `next`.

### CODE REVIEW

File: gitlab/__init__.py [hunk 1]
```diff
@@ -800,9 +800,9 @@ def current_page(self):
 
     @property
     def prev_page(self):
-        """The next page number.
+        """The prev page number.
```
Comment (nejch) [inline gitlab/__init__.py]:
```suggestion
        """The previous page number.
```

File: gitlab/base.py [hunk 2]
```diff
@@ -194,9 +194,9 @@ def current_page(self):
 
     @property
     def prev_page(self):
-        """The next page number.
+        """The prev page number.
```
Comment (nejch) [inline gitlab/base.py]:
```suggestion
        """The previous page number.
```

**Comment by donhui** [comment]
> Thank you @donhui, nice catch! And thanks for the commitlint fix :)
> 
> I just added 2 small suggestions (sorry for the nitpick). I think we can have the full word in the docs; as is done in https://docs.gitlab.com/ee/api/#other-pagination-headers.

Thank you, I have applied your suggestions.

**Comment by nejch** [comment]
That was quick, thanks! I've approved but Travis is currently down for maintenance, so we just have to wait for that to go green.

If you find any more inconsistencies, feel free to open a new PR :)

**Comment by donhui** [comment]
OK, thanks for your review.

**Pull request merged**